tomcat/src/main/tomcat-base/conf/server.xml

```diff
@@ -57,11 +57,7 @@
 
         &gcp-configuration;
 
-        <Context path="/">
-
-            &distributed-session-configuration;
-
-        </Context>
+        &distributed-session-configuration;
```

Comment:
It isn't modified, this is a revert to the original file (e.g the change is visible for the commit but not in the PR).